content/applications/marketing/surveys/create.rst

```diff
@@ -1,224 +1,331 @@
+required option.
+
+Below those fields and options are four tabs: :guilabel:`Questions`, :guilabel:`Options`,
+:guilabel:`Description`, and :guilabel:`End Message`.
 
 Questions tab
-=============
+-------------
+
+View, access, add, and/or delete questions and sections to the survey in the :guilabel:`Questions`
+tab.
 
-Add questions and sections to the survey in the :guilabel:`Questions` tab. A section divides the
-survey into parts in order to visually group similar questions together. To make a section, click
-:guilabel:`Add a section` and type in a section name. Then, add questions or drag and drop
-questions into the divided sections.
+By default, two columns are present in the :guilabel:`Questions` tab: :guilabel:`Title` (i.e. the
+question) and :guilabel:`Question Type`.
 
-Clicking :guilabel:`Add a question` opens the :guilabel:`Create Sections and Questions` pop-up to
-create and customize the survey question.
+If the :guilabel:`Randomized per Section` option is enabled in the :guilabel:`Options` tab of the
+survey form, a column titled, :guilabel:`# Questions Randomly Picked` appears in the
+:guilabel:`Questions` tab.
 
-.. image:: create/survey-question-pop-up.png
+Indicate whether questions require a mandatory answer by clicking the :guilabel:`(slider)` icon to
+the far-right of the column titles. Then, select :guilabel:`Mandatory Answer` from the drop-down
+menu to reveal the :guilabel:`Mandatory Answer` column in the :guilabel:`Questions` tab.
+
+.. image:: create/mandatory-answer-dropdown.png
    :align: center
-   :alt: The survey question pop-up window.
+   :alt: The slider drop-down menu with the Mandatory Answer option selected in Odoo Surveys.
 
-Create questions
-----------------
+Add a question
+~~~~~~~~~~~~~~
 
-In the :guilabel:`Create Sections and Questions` pop-up, type the question in the
-:guilabel:`Question` field. Then, choose the :guilabel:`Question Type`. A preview of how the
-question type looks is shown in the preview window.
+To add a question to a survey, click :guilabel:`Add a question` in the :guilabel:`Questions` tab,
+and proceed to fill out the :guilabel:`Create Sections and Questions` pop-up window that appears.
 
-Choose from the following :guilabel:`Question Types`:
+To learn how to create and customize questions, refer to our documentation on that.
 
-- :guilabel:`Multiple Lines Text Box`
-- :guilabel:`Single Line Text Box`
-- :guilabel:`Numerical Value`
-- :guilabel:`Date`
-- :guilabel:`Datetime`
-- :guilabel:`Multiple choice: only one answer`
-- :guilabel:`Multiple choice: multiple answers allowed`
-- :guilabel:`Matrix`
+.. important::
+   There **must** be a survey title entered in order for the :guilabel:`Create Sections and
+   Questions` pop-up window to appear. If no title is entered ofr the survey, an error pop-up
+   message appears in the upper-right corner, instructing the user to enter a survey title.
```

Comment:
```suggestion
   Questions` modal to appear. If no title is entered for the survey, an error pop-up message
   appears in the upper-right corner, instructing the user to enter a survey title.
```

Reply:
implementing everything *but* the "modal" suggestion 👍